SConstruct

```diff
@@ -779,6 +788,8 @@ noshell = has_option( "noshell" )
 
 jsEngine = get_option( "js-engine")
 
+rocksdb = get_option( "rocksdb" ) == "on"
```

Comment:
Our current code in v3.2 branch accepts `--rocksdb=1` option as a trigger to build with rocksdb support.
Is it possible to treat both `--rocksdb=1` and `--rocksdb=on` equivalently?